app/src/common/shared/org/mozilla/vrbrowser/ui/views/NavigationURLBar.java

```diff
@@ -340,6 +340,10 @@ public void handleURLEdit(String text) {
             if (!hasProtocol && !urlText.contains(" ") && UrlUtils.isDomain(urlText)) {
                 urlText = "https://" + urlText;
                 hasProtocol = true;
+            } else if (!hasProtocol && !urlText.contains(" ") && UrlUtils.isIPUri(urlText)) {
+                String protocol = UrlUtils.isLocalIP(urlText) ? "http://" : "https://";
+                urlText = protocol + urlText;
+                hasProtocol = true;
```

Comment:
I was thinking about ways we could degrade from https:// to http:// in FxR so we always try secure first.

Reply:
We can address that as a follow-up, we also need to handle that situation for normal URLs. Localhost and most of the local IPs are http, that's why I defaulted that way for now.